src/PrestaShopBundle/Install/XmlLoader.php

```diff
@@ -367,6 +367,56 @@ protected function getFallBackToDefaultEntityLanguage($iso, $entity)
         return file_exists($this->lang_path . $this->getFallBackToDefaultLanguage($iso) . '/data/' . $entity . '.xml') ? $iso : 'en';
     }
 
+    /**
+     * Special case for "country" entity.
+     */
+    public function populateEntityCountry()
+    {
+        $xml = $this->loadEntity('country');
+
+        // Read list of fields
+        if (empty($xml->fields)) {
+            throw new PrestashopInstallerException('List of fields not found for entity coutrny');
+        }
+        $langs = [];
+        $languageList = LanguageList::getInstance();
+        foreach ($this->languages as $id_lang => $iso) {
+            $languageList->setLanguage($iso);
+            $langs[$id_lang] = $languageList->getCountries();
+        }
+
+        // Load all row for current entity and prepare data to be populated
+        $i = 0;
+        foreach ($xml->entities->country as $node) {
```

Comment:
If you test `$xml->fields` should you test `$xml->entities->country` and be sure it's an array, no?

Reply:
Done ;)